src/refract/JSONSchemaVisitor.cc

```diff
@@ -211,6 +211,15 @@ namespace refract
             if (IsTypeAttribute(e, "nullable")) {
                 renderer.addSchemaType("null");
             }
+
+            if (e.value.second) {
+                IElement::MemberElementCollection::const_iterator defaultIt = e.value.second->attributes.find("default");
+
+                if (defaultIt != e.value.second->attributes.end()) {
+                    renderer.addMember("default", (*defaultIt)->clone());
```

Comment:
This works for primitive types, however for other types such as enum, array and object this falls down (see test examples).

The default value for primitive types are StringElement, BooleanElement, NumberElement which renders correctly. But in types where the default is not, it will be an ArrayElement etc. Where it will render as an object element.

Is there any easy way to solve this problem without creating some other visitor that will handle rendering the default element?